charts/devnet/templates/chains/virtual/eth/statefulset.yaml

```diff
@@ -0,0 +1,168 @@
+{{- range $chain := .Values.chains }}
+{{- if eq $chain.type "virtual" }}
+{{- if eq $chain.subtype "eth" }}
+
+{{ $defaultChain := get $.Values.defaultChains $chain.subtype | default dict }}
+{{ $chain = merge $chain $defaultChain }}
+
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: prysm
+spec:
+  serviceName: "prysm"
+  replicas: 1
```

Comment:
I guess currently we are not able to support multiple nodes for eth. Lets leave a todo here to make sure we are able to make this scalable down the line.